Stop exaggerating heading accuracy indicator size

friedbunny opened the pull request:
This change removes the 1.5× exaggeration of the user location annotation’s heading indicator.

Also cleaned up some of the heading mask calculations while I was at it.

Fixes #3718.

/cc @1ec5

friedbunny commented:
Unsure if this rises to changelog status. Probably? Haven’t added anything yet.

1ec5 commented:
> Unsure if this rises to changelog status.

Sure, why not. There’s visual impact to the user.

1ec5 commented:
👍

friedbunny commented:
Added a changelog entry.

I also realized that I hadn’t changed the maximum accuracy clipping size, so I went back and updated that to 20° (`90 - 70`). We now appear as accurate as Apple:

![img_4474](https://cloud.githubusercontent.com/assets/1198851/17905553/673fba70-6941-11e6-90f5-07d7c490aa92.PNG)

(Google doesn’t represent heading accuracy and uses a static heading arrow.)

friedbunny commented:
Note: I’m seeing reported accuracy as low as 10°, but we should still clip to 20° for display purposes.
